tests/test_util.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from pathlib import Path
+
+import pytest
+
+from mkosi.util import parents_below
+
+
+def test_parents_below_basic() -> None:
+    path = Path("/a/b/c/d/e")
+    below = Path("/a/b")
+    result = parents_below(path, below)
+    assert result == [Path("/a/b/c/d"), Path("/a/b/c")]
```

Comment:
I think the `result` is a bit superfluous and makes it easier to read, since it's a repeated line that doesn't do anything
```suggestion
    assert parents_below(path, below) == [Path("/a/b/c/d"), Path("/a/b/c")]
```
The tests that raise break this pattern.